### DISCUSSION TITLE
Sending notifcations via SMTP (EMAIL)

### DISCUSSION THREAD

**Discussion by ShadowofReason** [ParadoxAlarmInterface/pai#399] [Ideas]
Would it be possible for adding support for email notifications with the option to filter what events get sent out?

Mainly asking because I got the Email2SMS service from Telia, which I use to receive important/critical messages from other monitoring systems.

**Comment by yozik04** [comment]
Feel free to drop a Pull Request. There are some integrations already like pushover and pushbullet. I won't be able to invest time into it myself.
